src/main/java/org/folio/rest/impl/FinanceDataApi.java

```diff
@@ -40,7 +43,13 @@ public void getFinanceFinanceData(String query, String totalRecords, int offset,
   @Validate
   public void putFinanceFinanceData(FyFinanceDataCollection entity, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     financeDataService.putFinanceData(entity, new RequestContext(vertxContext, okapiHeaders))
-      .onSuccess(v -> asyncResultHandler.handle(succeededFuture(buildNoContentResponse())))
+      .onSuccess(financeDataCollection -> {
+        if (CollectionUtils.isEmpty(financeDataCollection.getFyFinanceData())) {
```

Comment:
I think better to provide the same response regadless of request body param for consistency.
So better to provide response for both Commit and Preview modes

Reply:
Updated it